src/layout/section/heroSection/story.tsx

```diff
@@ -21,9 +22,54 @@ stories.add('default', () => (
     )}
     heroImageUrlLarge={text(
       'heroImageUrlLarge',
-      'https://cdn.blablacar.com/kairos/assets/build/images/home_summer_campaign-1ea3207605913c1e26410e605a467eb7.jpg',
+      'https://cdn.blablacar.com/kairos/assets/build/images/home_summer_campaign_large-240e0d9fe2123506cca634c2acedce24.jpg',
     )}
     buttonText={text('buttonText', 'Rechercher un trajet')}
     buttonHref={text('buttonHref', 'http://google.fr')}
   />
 ))
+
+stories.add('search form', () => (
+  <MediaSizeProvider>
```

Comment:
I don't think you need it, since HereSection does not use it.

Reply:
And your are doing reponsive on the component itself with media queries

Reply:
The `SearchForm` component used in the story needs it.

Reply:
We already have a responsive component available on Storybook, no need to show it twice

Reply:
> We already have a responsive component available on Storybook, no need to show it twice
> @gabrielagabriel 

Which one?
The `SearchForm` component need the React provider in order to work correctly, otherwise only the CSS is updated (media query).

Reply:
Oh.. if its an adaptative layout you are right! 👍